### PR TITLE
Reimburse innis cafe

### PR DESCRIPTION
**Pull request by lwjohnst86**

### CODE REVIEW

File: treasurer/reimbursing/submitted/2017-02-06-innis-cafe.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+# Reimbursement form
+
+- Purpose of expense: Coffee and Tea for SWC workshop on Feb 2-3, 2017
```
Comment (QuLogic) [inline treasurer/reimbursing/submitted/2017-02-06-innis-cafe.md]: 3-4

File: treasurer/cheques/cheques.csv [hunk 2]
```diff
@@ -14,3 +14,4 @@ Number,Date,Recipient,Amount,Purpose,SignOff,Note,PR
 '013',2016-09-01,Luke Johnston,55.98,Snacks for Coder sessions,Luke and Lina,,38
 '014',2016-09-12,Luke Johnston,88.75,"Drinks for Fall term Hacky Hour, for GSU Pub",Luke and Lina,,43
 ‘015’,2016-10-05,Luke Johnston,48.92,Snacks for Coder sessions,Luke and Lina,,47
+‘018’,2017-02-06,Innis Cafe,409.63,Coffee and tea for the Feb 03 SWC workshops,Luke,,77
```
Comment (QuLogic) [inline treasurer/cheques/cheques.csv]: What happened to the other two cheques?

Comment (lwjohnst86) [inline treasurer/cheques/cheques.csv]: They are in other PRs (or will be soon)

File: treasurer/reimbursing/submitted/2017-02-06-innis-cafe.md (hunk 1 above)
Comment (lwjohnst86) [inline treasurer/reimbursing/submitted/2017-02-06-innis-cafe.md]: Right, thanks

**Pull request merged**